test/e2e/framework/metrics_util.go

```diff
@@ -53,7 +53,7 @@ const (
 
 	// We set a higher threshold for list apicalls as they can take more time when
 	// the list is really big. For eg. list nodes in a 5000-node cluster.
-	apiListCallLatencyThreshold time.Duration = 2 * time.Second
+	apiListCallLatencyThreshold time.Duration = 5 * time.Second
```

Comment:
Actually that raises one concern with the way the SLO is proposed:
- Suppose my cluster has Y+1 objects (just entering the 30s bracket).
- Now there's a regression increasing the latency by 5 times, but it will be still uncaught (as 25 < 30)

Maybe make the SLO linear instead of step function? Since we anyway have to go through the trouble of measuring count of the objects?